graph/flow/flow.go

```diff
@@ -0,0 +1,32 @@
+// Package flow provides control flow analysis functions.
+package flow
+
+import "gonum.org/v1/gonum/graph"
+
+// Graph is a control flow graph, a directed graph rooted at an entry node.
+type Graph interface {
+	// Entry returns the entry node of the control flow graph.
+	Entry() graph.Node
```

Comment:
`Root`?

Reply:
Sure, we can go for `Root` instead of `Entry`. In the literature I've mostly taken part of, the term *entry node* or *entry block* is used for control flow graphs. And the graph is a directed graph, rooted at the entry node.